vmm/src/memory_manager.rs

```diff
@@ -327,6 +327,9 @@ pub enum Error {
     #[cfg(target_arch = "aarch64")]
     /// Failed to create UEFI flash
     CreateUefiFlash(HypervisorVmError),
+
```

Comment:
This commit summary should be prefixed with `vmm:`

Reply:
Good catch. PR updated.